scripts/build_osm_network.py

```diff
@@ -370,6 +403,8 @@ def get_transformers(buses, lines):
         "bus0",
         "bus1",
         "voltage",
+        "voltage_bus0",
+        "voltage_bus1",
         "circuits",
         "length",
         "underground",
```

Comment:
@davide-f there are a couple of probably unnecessary columns available.
I would remove for now:
- length
- circuits
- tag_type
- bounds
- bus_0_coors + bus_1_coors

But keeping all these columns, for now, is probably not problematic. Meaning we could also deal later with it... How do you feel about it?

Reply:
Personally, I'd like to structure the csv files of transformers are they are supposed to be, similarly to pypsa-eur.
This is also to highlight possible issues and avoid possible problems down the line.

Reply:
Mh, I think carrying the geojson might be good for plotting and the voltage_bus0 and voltage_bus1 are good for choosing voltage-specific transformers. All other values can be dropped. But yeah the voltage can be also easy derived from matching with the bus list

Reply:
No on the voltages I agree, way easier to have them there, as well as the geometry; the rest could be dropped